CHANGELOG.md

```diff
@@ -5,12 +5,22 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Updated code for PureScript 0.14 (#120)
+- Removed `fold1Default` and deprecated `foldMap1Default` (#128)
```

Comment:
I believe this was the net result of breaking changes. Would be good to double-check.

Reply:
I think this is correct